config/environments/performance.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require Rails.root.join("config/environments/production")
+
+Rails.application.configure do
+  # Settings specified here will take precedence over those in config/environments/production.rb.
+
+  # Used to handle HTTP_X_WITH_SERVER_DATE header for server side datetime overwrite
+  config.middleware.use TimeTraveler
+
+  config.require_master_key = false
+
+  config.public_file_server.enabled = true
+  config.force_ssl = false
+  config.action_mailer.perform_deliveries = false
+  config.active_support.deprecation = :log
+  config.log_level = :debug
```

Comment:
do we need debug, as this might be noisy for logging and maybe we can lower (just to avoid getting pinged around too many logs from ops 😅 like we've gotten before)

Reply:
changed to depreciation :silence and log_level :fatal, seeing as you can't look at them in CI anyway